skipper.go

```diff
@@ -1266,16 +1278,12 @@ func listenAndServeQuit(
 			}()
 		}
 
-		if err := srv.ListenAndServeTLS("", ""); err != http.ErrServerClosed {
-			log.Errorf("ListenAndServeTLS failed: %v", err)
+		if err := srv.ServeTLS(l, "", ""); err != http.ErrServerClosed {
+			log.Errorf("ServeTLS failed: %v", err)
 			return err
 		}
 	} else {
 		log.Infof("TLS settings not found, defaulting to HTTP")
-		l, err := listen(o, o.Address, mtr)
```

Comment:
Will this break anything?

Reply:
listen() creates queuelistener if config is set, in this PR I move it out of "not HTTPS" branch to also use for HTTPS setup.

Reply:
I.e. for HTTP case nothing changes, for HTTPS queue listener will be used when config is specified.

You may also check stdlib for how srv.ListenAndServeTLS is different from srv.ServeTLS

Reply:
So it will also default, I didn't look close enough, thanks!